app/controllers/description_change_requests_controller.rb

```diff
@@ -8,8 +8,10 @@ def new
   def create
     @description_change_request = @planning_application.description_change_requests.new(description_change_request_params)
     @description_change_request.user = current_user
+    @current_local_authority = current_local_authority
```

Comment:
I don't think we need to set this as a variable. Line 41 can just call the method directly?